spec.html

```diff
@@ -25479,7 +25479,7 @@ <h1>Number.prototype.toExponential ( _fractionDigits_ )</h1>
             1. Let _d_ be `"0"`.
           1. Else,
             1. If _e_ &gt; 0, let _c_ be `"+"`.
-            1. Else _e_ &le; 0,
+            1. Else _e_ &lt; 0,
```

Comment:
`e` can't be equal `0` here: please see condition on line `25477`.

Reply:
in that case, can this be `else` with a separate `Assert`?